WordPress/Classes/Utility/Analytics/WPAppAnalytics+Media.swift

```diff
@@ -107,8 +113,28 @@ enum MediaUploadOrigin {
     case mediaLibrary(MediaSource)
     case editor(MediaSource)
 
-    func eventForMediaType(_ mediaType: MediaType) -> WPAnalyticsStat? {
+    // All new media tracking events will be added into WPAnalyticsEvent
+    // Ref: https://iosp2.wordpress.com/2020/03/16/adding-tracks-forget-wordpress-shared/
+    func eventForMediaType(_ mediaType: MediaType) -> WPAnalyticsEvent? {
+        switch (self, mediaType) {
+        // Media Library
+        case (.mediaLibrary(let source), .image) where source == .tenor:
+            return .mediaLibraryAddedPhotoViaTenor
+
+        // Editor
+        case (.editor(let source), .image) where source == .tenor:
+            return .editorAddedPhotoViaTenor
+
+        default:
+            return nil
+        }
+    }
+
+    // This is for the previous events created within WordPressShared
+    // Ref: https://iosp2.wordpress.com/2020/03/16/adding-tracks-forget-wordpress-shared/
+    func wpshared_eventForMediaType(_ mediaType: MediaType) -> WPAnalyticsStat? {
```

Comment:
Can this method name be more Swift-y? Like `wpsharedEventForMediaType` for example.

Reply:
Sure, that'd be a better name. Changed. Thanks @ScoutHarris